app/controllers/items/search_code.js

```diff
@@ -1,44 +1,112 @@
-import Ember from 'ember';
-import SearchCode from '../search_code';
-import AjaxPromise from 'stock/utils/ajax-promise';
-const { getOwner } = Ember;
+import Ember from "ember";
```

Comment:
Can we rename that page to something more meaningful ? e.g `assign_subform`

`search_code` is already (_wrongly_) used in many places, but I don't think we should accept that as a standard. As it suggests that this is a re-usable page used to search codes, which is not the case. It's a hardcoded page which does a very specific task. Which is why we have so many copy pasted pages.

I would normally suggest to make the page generic, but I know how deep we are in that bad practice, and refactoring it is a rough task. 

(FYI: I'm in the process of removing those search pages and replacing them with asynchronous animated overlays)